test/util/MessagesSpec.scala

```diff
@@ -17,31 +17,30 @@
 package util
 
 import play.api.i18n.MessagesApi
-import uk.gov.hmrc.play.test.{UnitSpec, WithFakeApplication}
 
 import scala.util.matching.Regex
 
-class MessagesSpec extends UnitSpec with WithFakeApplication {
+class MessagesSpec extends BaseSpec {
 
-  lazy val messagesAPI = fakeApplication.injector.instanceOf[MessagesApi]
+  lazy val messagesAPI = app.injector.instanceOf[MessagesApi]
```

Comment:
Would it be worth changing instances of app.injector.instandOf to injected given both are used throughout the project and do the same thing?

Reply:
Done